app/routers/kis_domestic_trading.py

```
@@ -0,0 +1,191 @@
+        async_result = celery_app.send_task("kis.execute_domestic_sell_orders")
+        return {
+            "success": True,
+            "message": "매도 주문이 시작되었습니다.",
+            "task_id": async_result.id
+        }
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=str(e))
+
+
+@router.post("/api/automation/per-stock")
+async def run_per_stock_automation(request: Request):
```

Ruff (0.14.6)

Unused function argument: `request`

(ARG001)